pandas/tseries/frequencies.py

```diff
@@ -607,7 +607,7 @@ def _is_quarterly(rule: str) -> bool:
 
 def _is_monthly(rule: str) -> bool:
     rule = rule.upper()
-    return rule in ("M", "BM")
+    return rule in ("M", "BM", "ME")
```

Comment:
should "M" be removed here?  for that matter, do we need to change BM to BME?

Reply:
good call, thanks!

Yeah we should be able to convert everything to the period alias before getting here, like
```diff
diff --git a/pandas/core/resample.py b/pandas/core/resample.py
index 407f15818b..2e4cb969c9 100644
--- a/pandas/core/resample.py
+++ b/pandas/core/resample.py
@@ -1631,7 +1631,7 @@ class PeriodIndexResampler(DatetimeIndexResampler):
         if is_subperiod(ax.freq, self.freq):
             # Downsampling
             return self._groupby_and_aggregate(how, **kwargs)
-        elif is_superperiod(ax.freq, self.freq):
+        elif is_superperiod(freq_to_period_freqstr(ax.freq.n, ax.freq.name), freq_to_period_freqstr(self.freq.n, self.freq.name)):
```

And then we can hopefully get rid of the extra 'ME' we've added to `_is_monthly` and `is_superperiod`

@natmokval do you want to try this?

Reply:
> do we need to change BM to BME

yup, as well as a few others. but to keep the scope down, I suggest we keep this one to just M -> ME, and then, once this is in, changing the others should just be a matter of updating the dict

https://github.com/pandas-dev/pandas/blob/c1beb9f2aa7bc5e5ed7593eb804cd7cade23ded9/pandas/_libs/tslibs/dtypes.pyx#L185-L187